packages/lab/src/table/Table.tsx

```diff
@@ -1,22 +1,23 @@
-import { clsx } from "clsx";
-import { ComponentPropsWithoutRef, forwardRef } from "react";
 import { makePrefixer } from "@salt-ds/core";
 import { useComponentCssInjection } from "@salt-ds/styles";
 import { useWindow } from "@salt-ds/window";
+import { clsx } from "clsx";
+import { type ComponentPropsWithoutRef, forwardRef } from "react";
 import "ag-grid-community/styles/ag-grid.css";
 import "@salt-ds/ag-grid-theme/salt-ag-theme.css";
 
 import tableCss from "./Table.css";
 
-export type TableProps = ComponentPropsWithoutRef<'table'> & 
-  {
-    variant?: "primary" | "secondary";
-  };
+export type TableProps = ComponentPropsWithoutRef<"table"> & {
+  variant?: "primary" | "secondary";
```

Comment:
zebra is also available :)

Reply:
how should rowspan be considered in this case?

Reply:
i think stick to what HTML will do if giving bg color to `tr`?